src/applications/check-in/components/LanguagePicker.jsx

```diff
@@ -1,32 +1,20 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import withTranslationEnabled from '../containers/withTranslationEnabled';
 
 function LanguagePicker() {
   const { i18n } = useTranslation();
```

Comment:
ah, this is much nicer than what I was doing :-)

Reply:
:)